Dockerfile

```diff
@@ -52,6 +54,7 @@ COPY --from=builder /go/src/github.com/makerdao/vdb-mcd-transformers/dockerfiles
 COPY --from=builder /go/src/github.com/pressly/goose/cmd/goose/goose goose
 COPY --from=builder /go/src/github.com/makerdao/vdb-mcd-transformers/db/migrations/* db/migrations/
 COPY --from=builder /go/src/github.com/makerdao/vdb-mcd-transformers/plugins/transformerExporter.so plugins/transformerExporter.so
+COPY --from=builder /go/src/github.com/makerdao/vulcanizedb/dockerfiles/wait-for-it.sh .
```

Comment:
probably want to copy this from `vdb-mcd-transformers`?

Reply:
oops yes, this probably would not build :hear_no_evil: Mind fixing?

Reply:
Fair enough - the reason this stood out to me is because I initially had trouble building this image against an older version of vdb, but I suppose that shouldn't be an issue moving forward.

Still feels a little odd adding the file to this repo and then not using it - any chance we'd want to modify it here without doing the same thing in vdb?

Thinking I may go ahead and rebase + either remove or reference the file, if that works for you?

Reply:
Yeah, feel free! Maybe let's use the file in this repo indeed then :)